tfx/components/example_validator/executor.py

```diff
@@ -31,12 +31,15 @@
 from tfx.types import artifact_utils
 from tfx.utils import io_utils
 
-
 # Key for statistics in executor input_dict.
 STATISTICS_KEY = 'statistics'
+
 # Key for schema in executor input_dict.
 SCHEMA_KEY = 'schema'
 
+# Key for training_statistics in executor input_dict.
+TRAINING_STATISTICS_KEY = 'training_statistics'
```

Comment:
nit: if these constants are only used in this module, consider privatize them.

For this and below.

Reply:
I privatized `_DEFAULT_FILE_NAME`, but `TRAINING_STATISTICS_KEY` and `ANOMALIES_KEY` are used in `executor_test.py` so I didn't privatize them.